Lib/test/test_sax.py

```diff
@@ -1573,5 +1573,17 @@ def test_all(self):
         check__all__(self, sax, extra=extra)
 
 
+class TestModule(unittest.TestCase):
+    def test_deprecated__version__and__date__(self):
+        for module in (sax.expatreader, sax.handler):
+            with self.subTest(module=module):
+                with self.assertWarnsRegex(
+                    DeprecationWarning,
+                    "'version' is deprecated and slated for removal in Python 3.20",
+                ) as cm:
+                    getattr(module, "version")
```

Comment:
With a string literal, getattr is not needed:

```suggestion
                    module.version
```

Reply:
Can do! Although it introduces an annoying PyCharm warning "Statement seems to have no effect", so perhaps?

```suggestion
                    assert module.version
```

Reply:
What if pycharm was right instead of wrong on this? 😈

Reply:
Or?:
```suggestion
                    _ = module.version
```

Reply:
I suggest keeping getattr(). It's uncommon to use getattr() to get an attribute, but it does what it says and prevents warnings such as the PyCharm warning.